docs/pages/api/framework/kedro.framework.context.md

```diff
@@ -18,3 +19,8 @@
 ::: kedro.framework.context.KedroContextError
     options:
       show_source: true
+
+::: kedro.framework.context.CatalogCommandsMixin
```

Comment:
As we already discussed, the styling here is weird https://kedro--4767.org.readthedocs.build/en/4767/pages/api/framework/kedro.framework.context/#kedro.framework.context.CatalogCommandsMixin but better to address all of these issues in one go